core/scripts/umip-3/2_VoteSimulate.js

```diff
@@ -36,9 +44,9 @@ async function runExport() {
   if (argv.repeated) {
     assert((await governor.numProposals()).toNumber() > 1);
   } else {
-    assert.equal((await governor.numProposals()).toNumber(), 1); // there should be 1 proposal on the first run.
+    assert.equal((await governor.numProposals()).toNumber(), 2); // there should be 1 proposal on the first run.
   }
-  assert.equal((await voting.getPendingRequests()).length, 0); // There should be no pending requests
+  assert.equal((await voting.getPendingRequests()).length, 1); // There should be no pending requests
```

Comment:
Same comment

Reply:
there should be no pending proposals. have updated accordingly